Sources/QuickObjectiveC/QuickSpec.m

```diff
@@ -26,7 +26,7 @@ @implementation QuickSpec
  included an expectation outside of a "it", "describe", or "context" block.
  */
 + (void)initialize {
-    [QuickConfiguration initialize];
+    [QuickConfiguration class];
```

Comment:
By accessing the class, let the Objective-C runtime call `+initialize` method of the class.